addons/website/static/src/core/website_edit_service.js

```diff
@@ -89,3 +90,24 @@ PublicRoot.include({
         websiteEdit.update(targetEl, options?.editableMode || false);
     },
 });
+
+// Patch Colibri.
+
+patch(Colibri.prototype, {
+    addListener(target, event, fn, options) {
+        fn = fn.bind(this.interaction);
+        // TODO No jQuery ?
+        const wysiwyg = $("#wrapwrap").data("wysiwyg");
+        const name = `${this.interaction.constructor.name}/${event}`;
+        let stealthFn = fn;
+        if (wysiwyg?.odooEditor) {
+            stealthFn = async (ev) => {
+                wysiwyg.odooEditor.observerUnactive(name);
+                const result = await fn(ev);
+                wysiwyg.odooEditor.observerActive(name);
+                return result;
+            };
```

Comment:
Shouldn't you also copy `stealthFn.isHandler = fn.isHandler`? (the goal is to keep the initial handler when you `refreshListeners` over and over)